pkg/sql/pgwire/pgcode/codes.go

```diff
@@ -56,6 +56,7 @@ var (
 	SQLserverRejectedEstablishmentOfSQLconnection = MakeCode("08004")
 	TransactionResolutionUnknown                  = MakeCode("08007")
 	ProtocolViolation                             = MakeCode("08P01")
+	ProxyConnectionError                          = MakeCode("08C00")
```

Comment:
is this a custom code for CRDB? if so, then i think we should put it below with the others: https://github.com/cockroachdb/cockroach/blob/03a5c4029931237dc204da82bacdca2385c4b2c7/pkg/sql/pgwire/pgcode/codes.go#L319

Reply:
oh yeah that's right. Thanks rafi.

Reply:
ok, I moved it.